tests/csrf_tests/tests.py

```diff
@@ -295,6 +347,35 @@ def test_token_node_with_new_csrf_cookie(self):
         csrf_cookie = resp2.cookies[settings.CSRF_COOKIE_NAME]
         self._check_token_present(resp, csrf_id=csrf_cookie.value)
 
+    def test_cookie_not_reset_on_accepted_request(self):
+        """
+        The csrf token used in posts is changed on every request (although stays equivalent).
+        The csrf cookie should not change on accepted requests. If it appears in the response,
+        it should keep its value.
+        """
+        req = self._get_POST_request_with_token()
+        CsrfViewMiddleware().process_view(req, token_view, (), {})
+        resp = token_view(req)
+        resp = CsrfViewMiddleware().process_response(req, resp)
+        csrf_cookie = resp.cookies.get(settings.CSRF_COOKIE_NAME, None)
+        if csrf_cookie:
+            self.assertEqual(csrf_cookie.value, self._csrf_id_cookie,
```

Comment:
Not sure what you mean; if you think line 362 needs to be indented more than 363, I disagree. If you mean something else, please elaborate.